proprietary/Configure-TI-CC2541-compatible-proprietary-PHY.md

```diff
@@ -0,0 +1,134 @@
+<details>
+<summary><font size=5>Table of Contents</font> </summary>
+
+- [1. Introduction](#1-introduction)
+- [2. Prerequiesite](#2-Prerequisite)
+- [3. PHY Confirm](#3-PHY-Confirm)
+- [4. EFR32xG22 project](#4-EFR32xG22-project)
+- [5. Conclusion](#5-Conclusion)
+
+</details>
+
+# 1. Introduction
+Sometime we got request on configure compatiable PHY for communicate with device from other vendor. Here share the experiece on configure an EFR32xG22 PHY communicate with TI CC2541 [proprietary 2.4G PHY](files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/2M_BS-TX_2.xml). 
+To configure the PHY, we need to know the detail parameter on radio configurator.
+
```

Comment:
It recommends to replace "we" to "it" in the article.
To configure the PHY, it needs to know the detail parameters of radio configurator.

Reply:
agree